README.md

```diff
@@ -1,117 +1,174 @@
-# Secure Transport
+#### Secure Transport
 
-## Prerequisites
+#### Prerequisites
 
-- Docker version >= 17.11
-- Docker-compose version >= 1.17.0
+- Docker version >= 19.X
+- Kubernetes >= 1.9 or Docker-compose version >= 1.17.0
 
 #### Basic parameters are configurable through environment variables and files
 
```

Comment:
The github repo should contain full blown yml files to serve as working examples - the details are in the Deployment Guide. For example - the guide says tha tthe Edge requires external mysql db. That's it. In the Github repo we can put a k8s yml file to deploy mysql single node db for playing with the solution.
Ful haproxy.cfg files should also be part of the github repo - only the most important required parts related to haproxy must be docuemtned in the deployment guide.